src/sql/parse.py

```diff
@@ -45,6 +45,13 @@ def parse(cell, config):
         "return_result_var": False,
     }
 
+    select_pointer = cell.lower().find("select")
```

Comment:
it'll be unreliable to rely on "select". for example, in DuckDB, this are the same: `select * from table` and `from table`. In general, I'd avoid making any assumptions about the user's query.

perhaps there's a better way to parse it? for example, you can look at the first occurrence of `<<` (I believe we're already doing that)